packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -128,6 +131,16 @@ const nextServerlessLoader: loader.Loader = function() {
       export default async (req, res) => {
         try {
           await initServer()
+
+          ${
+            basePath
+              ? `
+          if(req.url.startsWith(${basePath})) {
```

Comment:
```suggestion
          if(req.url.startsWith('${basePath}')) {
```
Looks like these need quotes still